### PR TITLE
Assignment6

### PR DESCRIPTION
**Pull request by OkayNiceBrian**

### CODE REVIEW

**Comment by MeguminXIII** [comment]
I will review

File: README.md [hunk 1]
```diff
@@ -4,6 +4,9 @@
 ## Overview
 
 In this project, we are going to configure continuous integration and continuous deployment.
+Link to branch: https://github.com/OkayNiceBrian/EWU-CSCD379-2021-Spring/tree/Assignment6
+Link to site: https://briansecretsanta.azurewebsites.net/
+![](../../workflows/.NET/badge.svg)
```
Comment (MeguminXIII) [inline README.md:9]: You need to swap .NET with the -name of your .yml, I had to make the -name of mine 1 word as well in order to get it to work. the -name is the first line of your AssignmentBuild.yml file

So as an example currently the name would be: Assignment Build And Test
What I did was remove the spaces and then put it where you have the .NET in, Alternatively you could try and enclose it in quotation marks but I am unsure if that would work ex:

or as one word:

**Review by MeguminXIII — COMMENTED** [review]
Assignment
Configure a CI Build against the SecretSanta solution
Compile all the source code ✔
Run all the user tests ✔
Add the Build Status to the Read.me file of your project. (Be sure to use relative links so it works across branches and even when forked.) ❌

So it appears that you have a .yml Assignment build, tho it isn't showing up in the pr for some reason, I had to go into your branch to see it, 
in your Readme your badge isn't showing

Place a URL to your fork and branch in your readme (so that we can easily navigate there to see the build)✔
Configure a CD deployment of the project (https://docs.microsoft.com/en-us/azure/app-service/deploy-github-actions)✔
Configure an Azure Web App (using the Azure Portal) ✔
Configure a GitHub Action to deploy to an Azure Web App (https://github.com/Azure/webapps-deploy) ✔
Add the Deploy Status to the Read.me file of your project. (Be sure to use relative links so it works across branches and even when forked.) ❌

Can do this via a badge, Example: /badge.svg), I think you need to make it 1 word, but you may be able to enclose it in quotation marks? I haven't tested that

Be sure to deploy the API project and verify that the UI that calls the API project works correctly. This will require you to update the URL/address in the Web project that points to the API project. ❌
Have not done this

Place a hyperlink to the running website in both your Readme and your PR. (Consider using a badge that shows whether the site is up or not.) ❌
It is not in your PR, it would go in the conversation tab, at the top is an empty comment that you can edit it into.

Extra Credit
Have the Web Tests run against the deployed website after the deployment completes.❌
Configure the API site URL address into the Web Project during the deployment using a GitHub Action variable.❌
Configure code coverage and generate a report (see https://github.com/dotnet/docs/blob/main/docs/core/testing/unit-testing-code-coverage.md) ✔❌

Looks good just need to deploy API and fix a badge and add a badge to the Readme and the hyperlink to the pr, and a badge to show if the website is running in both as well since the specifications mention it.

**Comment by mmwoodfo** [comment]
The code review that was done by MeguminXIII covers what you are missing.

**Pull request closed without merging**